plugins/monorepo/src/tasks/workspace-command.ts

```diff
@@ -0,0 +1,44 @@
+import { Task, TaskRunContext } from '@dotcom-tool-kit/base'
+import { runTasksFromConfig } from 'dotcom-tool-kit/lib/tasks'
+import { WorkspaceCommandSchema } from '@dotcom-tool-kit/schemas/lib/tasks/workspace-command'
+import {minimatch} from 'minimatch'
+import pluralize from 'pluralize'
+import { styles } from '@dotcom-tool-kit/logger'
+import LoadWorkspaceConfigs from '../load-workspace-configs'
+
+
+export default class WorkspaceCommand extends Task<{ task: typeof WorkspaceCommandSchema }> {
+  async run({ command, files }: TaskRunContext) {
+    const configsWithCommand = LoadWorkspaceConfigs.configs.filter(
+      ({config}) => command in config.commandTasks
```

Comment:
**question**: shouldn't this be checking for `this.options.command` if that's defined? likewise for other uses of `command` in this file